sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -2673,6 +2700,35 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				currentVersion = 309;
 				getDao(Event.class).executeRaw("ALTER TABLE events ADD COLUMN eventIdentificationSource varchar(255);");
 
+			case 310:
+				currentVersion = 310;
+				//@formatter:off
+				getDao(VaccinationEntity.class).executeRaw(
+					"CREATE TABLE vaccination (\n"
```

Comment:
* Remove all occurrences of `\n`
* Please remove any not null constraints here (missed to put that in the issue description) because not null constraints are a huge problem in our SQLite version (mainly because we can't remove them afterwards)
* SQLite expects column names being written in camel case; please change all column names accordingly